sdk/python/feast/repo_config.py

```diff
@@ -175,6 +180,12 @@ class RepoConfig(FeastBaseModel):
     def __init__(self, **data: Any):
         super().__init__(**data)
 
+        if "registry" in data and "type" in data["registry"]:
+            if data["registry"]["type"] == "snowflake.registry":
+                self.registry = get_registry_config_from_type(data["registry"]["type"])(
+                    **data["registry"]
+                )
```

Comment:
Sorry I'm not sure I understand this - does this mean that the repo config object is also going to have a reference to the registry, but only in case of snowflake? What about other types? And is there a reason why this reference needs to exist here specifically? Seems like it isn't being used anywhere?

Reply:
I refactored this so they're all consistent